karabo_bridge/client.py

```diff
@@ -58,11 +59,10 @@ class Client:
     ZMQError
         if provided endpoint is not valid.
     """
-    def __init__(self, endpoint, sock='REQ', ser='msgpack', timeout=None):
+    def __init__(self, endpoint, sock='REQ', context=None, timeout=None):
```

Comment:
Just thought of an extra one: if any code is passing `ser='msgpack'`, this will break it. Should we accept the parameter for now but throw an error if it's anything besides `'msgpack'`?

Also, if we're being strict about API compatibility, new parameters (`context=`) should only be added at the end of the list, to avoid breaking code that passes parameters positionally. We could use keyword-only arguments to limit this problem for the future.

Reply:
Fine with that.